kotlin/samples/hello-workflow/src/main/java/com/squareup/sample/helloworkflow/HelloLayoutRunner.kt

```diff
@@ -15,25 +15,12 @@
  */
 package com.squareup.sample.helloworkflow
 
-import android.view.View
-import android.widget.TextView
-import com.squareup.workflow.ui.ContainerHints
-import com.squareup.workflow.ui.LayoutRunner
-import com.squareup.workflow.ui.LayoutRunner.Companion.bind
+import com.squareup.sample.helloworkflow.databinding.HelloGoodbyeLayoutBinding
 import com.squareup.workflow.ui.ViewBinding
+import com.squareup.workflow.ui.bindViewBinding
 
-class HelloLayoutRunner(view: View) : LayoutRunner<HelloWorkflow.Rendering> {
-  private val messageView: TextView = view.findViewById(R.id.hello_message)
-
-  override fun showRendering(
-    rendering: HelloWorkflow.Rendering,
-    containerHints: ContainerHints
-  ) {
-    messageView.text = rendering.message
-    messageView.setOnClickListener { rendering.onClick() }
+val HelloBinding: ViewBinding<HelloWorkflow.Rendering> =
+  bindViewBinding(HelloGoodbyeLayoutBinding::inflate) { rendering, _ ->
+    helloMessage.text = rendering.message
+    helloMessage.setOnClickListener { rendering.onClick }
```

Comment:
Should we leave this sample using the `View`-based `LayoutRunner`, make it use a binding-based `LayoutRunner`, or show this most succinct style?

Reply:
I think the only one might be the screen in the dungeon sample where we initialize the recycler view.

Reply:
I converted a bunch of samples, but wow are there a lot. Need to timebox this for now. We shouldn't merge this until all samples are converted though, so it's not confusing what the "right" way to do things is.